README.md

```diff
@@ -19,10 +19,9 @@ For features _outside_ the scope of winit, see [Are we GUI Yet?](https://arewegu
 
 ## Contact Us
 
-Join us in any of these:
+Join us in our [![Matrix](https://img.shields.io/badge/Matrix-%23rust--windowing%3Amatrix.org-blueviolet.svg)](https://matrix.to/#/#rust-windowing:matrix.org) room. If you don't get an answer there, try [![Libera.Chat](https://img.shields.io/badge/libera.chat-%23winit-red.svg)](https://web.libera.chat/#winit).
```

Comment:
@kchibisov: Why do we have both a Matrix and the Libera chat? Personally, I'm basically never active on the latter, but does it see use? Or should we deprecate it?

Reply:
it's because the bridge died, I used it all the time and still present there, maybe if bridge resurrects.

Reply:
Ah yeah, the IRC bridge, I thought we just sorta gave up on IRC when that happened. But if you're still active on Libera.Chat, and if people use it, then I'm fine with keeping the link.

Reply:
Would prefer to remove it because not all maintainers are there, but all maintainers are on Matrix.

Reply:
yeah, but some people can't be bothered to create matrix account to ask question, so I'm not sure how to _solve_ that part. With irc you just ask and go, though, it's kind of not that active right now.